_commands/scripting/appendOperator.md

```diff
@@ -0,0 +1,57 @@
+---
+---
+
+>\>
+-----
+
+'>>' is a shell operator that you can use to append the output in an existing file.
+
+~~~ bash
+$ echo line 1 >> example.txt
+$ cat example.txt
+line 1
+$ echo line 2 >> example.txt
+$ cat example.txt
+line 1
+line 2
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+If the file does not exist, '>>' creates the file.
+
+~~~ bash
+$ echo line 1 >> example.txt
+$ cat example.txt
+line 1
+~~~
+
+The appending operator allows for the appending of standard output and standard error to a file.
+
+Appending stdout to the file is appending the output of a file to the end of the other file.
+
+~~~ bash
+$ python random.py >> output.txt
+Traceback (most recent call last):
+  File "random.py", line 2, in <module>
+    print 3 / 0
+ZeroDivisionError: integer division or modulo by zero
+$ cat output.txt
+Random text
```

Comment:
I'd like an example where you 'cat' the file before and after appending text to it. Also, please us a different program to generate text rather than python (see the reference page for examples). It's not intuitive for a new reader to understand what's going on with the python script, and the goal is to demonstrate usage of the '>>' operator and not python.